config/routes.rb

```diff
@@ -16,4 +16,15 @@
     end
   end
 
+  #API
+  namespace :api, defaults: {format: 'json'} do
+    namespace :v0 do
+      resources :users, only: :show do
+        resources :posts, only: [:index, :show], shallow: true do
+          resources :comments, only: [:index, :create]
+        end
+      end
```

Comment:
- [Optional] Hello Team. Great job 🙌, Just a quick suggestion though. I notice you have used `shallow: true` which makes your nested route a bit different. I believe using `shallow: true` on this occasion is not really necessary. This is because your route will make better sense if only be nested inside the user/post. 

> something like 
```ruby
/api/v0/users/:user_id/posts/:post_id/comments
```